manage.py

```diff
@@ -186,6 +199,19 @@ def create_user(email, role):
     db.session.add(user)
     db.session.commit()
 
+def __parse_tech_names__(tech_str):
+    if tech_str == 'all':
+        return watcher_registry.keys()
+    else:
+        return tech_str.split(',')
+
+def __parse_accounts__(account_str):
```

Comment:
See the underscore comment for `def __parse_tech_names__(tech_str):`.

Same thing applies here.